packages/feedback/src/environments/environment.prod.ts

```diff
@@ -0,0 +1,8 @@
+export const environment = {
+  production: true,
+  name: '',
+  api: '',
+  graphqlAPI: '',
+  client: '',
+  subscription: ''
```

Comment:
`authorization` is missing from this environment file.

Reply:
Also, this file should only contain the fields that are being used in the code.

If it's not being used, don't define it here.
And all the environment files should have the same fields. The values can be different.

Reply:
Okay